serving/backend/server.py

```diff
@@ -24,18 +22,9 @@
 model = joblib.load(filename="./saved_models/current_model.pkl")
 
 
-DATASETS_DIR = "pathfinder_2e_data"
-DATASET_FILES = [
-    "pathfinder-bestiary.db",
-    "pathfinder-bestiary-2.db",
-    "pathfinder-bestiary-3.db",
-]
-DATASET_PATHS = [f"{DATASETS_DIR}/{file}" for file in DATASET_FILES]
+DATASET_PATH = "./counterfactuals_datasets/bestiaries_basic.csv"
 
-df = load_and_preprocess_data(
-    DATASET_PATHS,
-    characteristics=ORDERED_CHARACTERISTICS,
-)
+df = read_csv(DATASET_PATH)
```

Comment:
To już tak przy okazji - df jako zmienna na poziomie modułu też może być capslockiem